addons/mil_ato/fnc_ATO.sqf

```diff
@@ -1131,7 +1131,7 @@ switch(_operation) do {
             private _sidesEnemy = [];
 
             {
-                if ((_side getfriend (call compile _x)) < 0.6) then {
+                if ((_side getfriend (missionNamespace getVariable [_x, "WEST"])) < 0.6) then {
```

Comment:
getFriend takes SIDE. Your default value is STRING, that won't work out. Maybe this should be the stringToSide function thing?

Reply:
Yes that would work but do replace `"RESISTANCE"` in the `_sides` array with `"GUER"` if you do that.

Reply:
Shouldn't that handling better be moved into the stringToSide func?

Reply:
```sqf
private _sides = [east, west, resistance];

{
                if ((_side getfriend (missionNamespace getVariable [_x, west])) < 0.6) then {
                    _sidesEnemy pushBack _x
                };
            } foreach (_sides - [_side]);
```

Reply:
> Shouldn't that handling better be moved into the stringToSide func?
Well perhaps but `"RESISTANCE"` shouldn't really exist anyway since `str resistance` results in `"GUER"`. Ideally all these side thingies should go through the proper functions that handle it. I've only found `"RESISTANCE"` in this file and `fnc_OPCOM.sqf` so it's probably better to just change them all to `"GUER"`. Hoping it won't affect anything else.